supervised/algorithms/knn.py

```diff
@@ -73,7 +81,7 @@ def __init__(self, params):
         )
 
 
-class KNeighborsRegressorAlgorithm(KNNFit):
+class KNeighborsRegressorAlgorithm(KNNFit, RegressorMixin):
```

Comment:
we derive from `KNNFit` which derives from `ClassifierMixin` and we also derive from `RegressorMixin`. This looks strange ...